omnibus/files/private-chef-cookbooks/private-chef/recipes/postgresql-external.rb

```diff
@@ -20,4 +20,5 @@
   include_recipe "private-chef::erchef_database"
   include_recipe "private-chef::bifrost_database"
   include_recipe "private-chef::oc_id_database"
+  include_recipe "private-chef::bookshelf_database"
```

Comment:
Are we going to create this regardless of whether we've configured the storage to be `:sql`?